### ISSUE TITLE
Support for additional catalogs

### ISSUE THREAD

**Issue by kfox1111** [operator-framework/operatorhub.io#292]
We have some automation that would make it easy to maintain a catalog container automatically with our own operators instead of manually creating PR's against https://github.com/operator-framework/community-operators. Could the website add support for additional catalogs?

**Comment by dmesser** [comment]
We are looking at future options to allow the site to populate content from multiple catalogs. It's kind of blocked by the [community-operators](https://github.com/operator-framework/community-operators) supporting references to other catalogs.

There is a balance we need to strike between being open enough and maintain some level of average quality so the hub isn't full of duplicates of MyCoolOperator 0.0.1.

The other thing we are working in is self-sufficient PR merging based on `CODEOWNERS`.

Happy to discuss this in our [monthly community meeting](https://github.com/operator-framework/community/projects/2)!
